charts/kubernetes-agent/templates/pod-roles.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "kubernetes-agent.scriptPodWorkerRoleName" . }}
```

Comment:
Is this always deployed? We may want to consider allowing users to override this.

Reply:
Good point - I've wrapped this in a "if worker is enabled" - is that the cleanest way to protect against deploying an unwanted role?

Reply:
For this, yeah. Is this role unable to be defined by the user at all?

Reply:
I think no - for now, its implicit.